front/public/locales/en/rollingstock.json

```
@@ -22,10 +22,11 @@
   "electricOnly": "Electric only",
   "equipement": "Equipment",
   "errorMessages": {
-    "missingInformation": "Missing information",
-    "must be number": "Number is required",
-    "must be equal to one of the allowed values": "Authorised value required",
-    "rollingStockUsed": "Rolling stock is used in the following scenarios:",
+    "missingInformation": "missing informations",
+    "must be number": "must be a number",
```

We use camelCase for the keys (same for the french file)
```suggestion
    "mustBeNumber": "must be a number",
```

---

it was for ajv resolver errors, I have to remove them